lib/lambda_ethereum_consensus/beacon/sync_blocks.ex

```diff
@@ -12,7 +12,7 @@ defmodule LambdaEthereumConsensus.Beacon.SyncBlocks do
   alias LambdaEthereumConsensus.P2P.BlockDownloader
   alias LambdaEthereumConsensus.StateTransition.Misc
 
-  @blocks_per_chunk 20
+  @blocks_per_chunk 16
```

Comment:
why this change?

Reply:
It just bothers me that it isn't a power of 2 😅